### PR TITLE
Added changes in Script.js file.

### PR DESCRIPTION
**Pull request by archana-9430**
Added the details of Simple Interest Calculator project.

## Related Issue
  - Info about Issue or bug

Closes: #[issue number that will be closed through this PR]

### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

### Checklist:

- [x] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
 let projectData = [
   {
+    projectName: "Interest Calculator",
```
Comment (Vishal-raj-1) [inline script.js:3]: add ProjectImage as well

Comment (archana-9430) [reply script.js:3]: How to add ProjectImage ?

Comment (991rajat) [reply script.js:3]: Create a gif of your project save it in assets/gif and
add the link in script.js

**Comment by Vishal-raj-1** [comment]
@archana-9430 updates ?

**Comment by archana-9430** [comment]
> @archana-9430 updates ?

In Progress...

**Comment by archana-9430** [comment]
> @archana-9430 updates ?

@Vishal-raj-1 , do check PR #977 and #979.

**Comment by 991rajat** [comment]
@archana-9430 This PR is not required right?

**Comment by 991rajat** [comment]
As you have created another PR. I'm closing this

**Pull request closed without merging**